### ISSUE TITLE
Edge support

### ISSUE THREAD

**Issue by wynfred** [digitalascetic/ngx-pica#12]
Hello, MS Edge doesn't support the File constructor. Could you please change the `blobToFile` function and use a workaround there?

**Comment by corcorb** [comment]
This appears to also affect iOS when running in a WebView as well (e.g., via Cordova). Running in Safari on iOS works fine. Tested in iOS 12.1 on an actual device and on a simulator.

I've worked around it locally by using my own copy of the library with `blobToFile` updated to be:

```typescript
  private blobToFile(blob: Blob, name: string): File {
    const formData = new FormData();
    formData.set('file', blob, name);
    return formData.get('file') as File;
  }
```

I'll create a PR to that effect in case that's the direction the team wants to go.

**Comment by wynfred** [comment]
I have a fork that fixed this issue using a different approach. [here](https://github.com/wynfred/ng7-pica/blob/master/projects/ngx-pica/src/lib/ngx-pica.service.ts#L253)
An exif rotation bug from compressImage was also fixed there.

**Comment by corcorb** [comment]
@wynfred your approach looks a lot less hacky than my `FormData`.
If you wanted, you could combine that to a one-liner:

```typescript
return Object.assign(new Blob([blob], { type: type }), { name: name, lastModified: lastModified });
```

**Comment by edezacas** [comment]
@corcorb PR are welcome 😄

**Comment by corcorb** [comment]
PR created: https://github.com/digitalascetic/ngx-pica/pull/17

**Comment by edezacas** [comment]
Released 7.0.3. This bug should be fixed.

**Comment by corcorb** [comment]
Many thanks for the quick release. I've tested it and it resolves the issue I was seeing.
